### ISSUE TITLE
Add new method to Helpers.Scroll decorator

### ISSUE THREAD

**Issue by dmytro-lymarenko** [fisshy/react-scroll#147]
I suggest to add a method to Helpers.Scroll decorator:
```
scrollToThis : function() {
    this.scrollTo(this.props.to, this.props);
}
```
It allows us to do the next:
when you create links you can store the ref of the links and then when you need you can call this function: ref_to_link.scrollToThis();
In my project I had to scroll to the Element when page is downloaded (Container is mounted) and I used this ways: ref_to_link.scrollTo(ref_to_link.props.to, ref_to_link.props); And if we have the method I recommend to add we can write it shortly:
ref_to_link.scrollToThis();

**Comment by zeroasterisk** [comment]
I agree - I would like a programatic interface to scroll to various "anchors" on the page based on events which are not link-clicks.  _(example: when an item updates from a long running / async datasource, we may want to scroll to it)_

**Comment by fisshy** [comment]
Great suggestion, feel free to make a PR :)
